src/annotations/items/AbstractAnnotationLine.cpp

```diff
@@ -57,7 +57,7 @@ QLineF AbstractAnnotationLine::line() const
 	return *mLine;
 }
 
-void AbstractAnnotationLine::setPointAt(const QPointF &point, int index)
+void AbstractAnnotationLine::setPointAt(const QPointF &point, int index, bool keepAspectRatio)
```

Comment:
For line it's not really aspectRatio but snapping to angles, maybe we could implement that here too?